### ISSUE TITLE
extended hours info

### ISSUE THREAD

**Issue by eliwelch17** [jmfernandes/robin_stocks#391]
RH now allows everyone to do extended hours trading, but only for a list of 43 stocks. Will the ability to trade in extended hours bool be aded to the dict returned by get_fundamentals function?

**Comment by mw66** [comment]
Based on #435 it's possible to place the order, have you tried?

BTW, is it possible to get the extended hour stock price?

And what are the 43 stock list?

**Comment by doctorcolossus** [comment]
@mw66, yes: in each dict in the list returned by `robin_stocks.robinhood.get_quotes`, you should see keys called `"last_extended_hours_trade_price"` and `"last_non_reg_trade_price"`, respectively containing the extended hours and 24-hour quotes.
